### PR TITLE
Upgrade decidim 0.15.2

### PR DESCRIPTION
**Pull request by aitorlb**
- Update `decidim` version `'0.15.2'`
- Update tags:
  - `gem 'decidim-cas_client', tag: 'v0.0.17'`
- Run migrations
- Add upgrade notes
- Add whenever gem

**Pending:** testing `decidim-cas_client` with new decidim version

### CODE REVIEW

File: docs/UPGRADE.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+Update your Gemfile:
+```
+gem "decidim", "0.XX.X"
+gem "decidim-dev", "0.XX.X"
+```
+
+Run these commands to upgrade and make sure you get all the latest migrations:
+```
+bundle update decidim
```
Comment (tramuntanal) [inline docs/UPGRADE.md]: Great idea to add this document @aitorlb 
Just note that this should be executed locally, and you are not metioning the git commands . I would add the git commands and explain it here with a notice.
I'll also add a new section for what should be executed in production that is only the `db:migrate`.
```suggestion
# to be executed locally on the developer's machine
bundle update decidim
bin/rails decidim:upgrade
git add db/migrations
git commit db/migrations
git push
bin/rails db:migrate

# to be executed in the deployment environments
bin/rails db:migrate
```

Comment (aitorlb) [reply docs/UPGRADE.md]: It was @isaacmg410 idea. Thanks for the tips.

File: config/schedule.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+#
+# every 4.days do
+#   runner "AnotherModel.prune_old_records"
+# end
+
+# Learn more: http://github.com/javan/whenever
+
+every 1.day, at: '2:00 am' do # 1.minute 1.day 1.week 1.month 1.year is also supported
+  rake "decidim:metrics:all"
+end
```
Comment (isaacmg410) [inline config/schedule.rb]: Add data_portability rake task, please.

Comment (aitorlb) [reply config/schedule.rb]: Thanks

**Pull request merged**